step2-01/README.md

```diff
@@ -2,54 +2,81 @@
 
 [Lessons](../) | [Exercise](./exercise/) | [Demo](./demo/)
 
-In this step, we'll cover enough of the TypeScript concepts to be productive with the React & Redux frameworks.
+In this step, we'll cover enough TypeScript concepts to be productive with the React & Redux frameworks.
```

Comment:
Might be good to have a subheading about *why TypeScript?* Or at least something to mention in the lecture. Some points that come to mind:
- Live in the future now! Use new ES constructs that aren't supported by all browsers yet (looking at you IE 11...).
- Type safety can help make projects more maintainable and self-documenting